Tools/Scripts/run-gtk-tests

```diff
@@ -91,6 +91,9 @@ class GtkTestRunner(TestRunner):
     def is_wpe_platform_wayland_test(self, test_program):
         return False
 
+    def is_openxr_test(self, test_program):
+        return "Immersive" in os.path.basename(test_program)
+
```

Comment:
Since this function is the same fot GTK and WPE it can be moved to the base class `TestRunner` in Tools/glib/api_test_runner.py